CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 ### New features
 - Add a new feature: Trace Profiling. See more details about it on our [website](http://kindling.harmonycloud.cn). ([#335](https://github.com/CloudDectective-Harmonycloud/kindling/pull/335))
 
+### Bug fixes
+- Fix the bug of incomplete records when threads arrive at the cpu analyzer for the first time
+
```

Comment:
Please put this part of change under the section `Unreleased`.

Reply:
ok，CHANGLOG.md has just been updated.